measurement/validation.py

```diff
@@ -33,18 +36,25 @@ def get_data_to_validate(
     start_time_ = datetime.strptime(start_time, "%Y-%m-%d").replace(tzinfo=tz)
     end_time_ = datetime.strptime(end_time, "%Y-%m-%d").replace(tzinfo=tz)
     extra = {}
-    if not include_validated:
+    if only_validated:
+        extra = {"is_validated": True}
+    elif not include_validated:
         extra = {"is_validated": False}
```

Comment:
I think that in this new approach, we are showing either only validated data or only not validated data. It is not possible - or even desirable because of the confusion - to display all the data, validated or not. Therefore, a single flag (eg. `is_validated`) to control what should be shown is enough. That also makes the use of `extra` unnecessary since the flag can be used directly in the filter..